src/core_seaice/Registry.xml

```diff
@@ -639,6 +659,10 @@
 			description="Run the ridging column physics calculation."
 			possible_values="true or false"
 		/>
+		<nml_option name="config_use_column_snow" type="logical" default_value="false" units="unitless"
```

Comment:
This name is ambiguous. Snow is used whenever column physics is used so what is really being switched on here?